lib/static/components/state/state-error.jsx

```diff
@@ -46,6 +56,24 @@ class StateError extends Component {
         return null;
     }
 
+    _tryToImproveSnippet(snippet) {
```

Comment:
By default, `_errorToElements` slices our string into lines, and picks first one to be a title.

So at first it looked like this:

<img width="899" alt="image" src="https://github.com/gemini-testing/html-reporter/assets/54893992/098750ef-fa80-4df7-a186-e3378a90dd26">

Here i am trying to pull out "filename" and use it as first line, so it would became title:

<img width="889" alt="image" src="https://github.com/gemini-testing/html-reporter/assets/54893992/52780608-70af-4312-95a6-09aeed4dfada">